safe/gui/tools/impact_merge_dialog.py

```diff
@@ -546,6 +556,28 @@ def validate_all_layers(self):
                         'Keyword aggregation attribute not found for '
                         'aggregation layer.'))
 
+    def normalize_dict_keys(self, input_dict):
+        """Normalizing to force keys to ascii.
+
+        Possible bugs: This function can't handle if we had two or more keys
+        with the same normalized form. It will collide. The next keys will
+        override previous keys.
+
+        Example, if somehow we had two keys 'District a' and 'District b',
```

Comment:
No, that way it becomes District and Dstrict. I want to use french accent for a and b, but I'm just too lazy to find the character map :p